src/actions/portfolio/handlers/quantum/index.js

```diff
@@ -42,7 +42,7 @@ export const getQuantumPortfolio = async (
     }
     portfolio.push({
         symbol: "QTUM",
-        balance: getBtcFromSatoshis(new Decimal(balance).toFixed()),
+        balance: getBtcFromSatoshis(new Decimal(balance)),
```

Comment:
The toFixed is required for consistency. Every portfolio is saved with a string balance.

Reply:
I moved the `toFixed()` inside the `getBtcFromSatoshis`, after the `dividedBy`.